src/mono-runtimes/mono-runtimes.targets

```diff
@@ -545,6 +545,8 @@
     />
     <ItemGroup>
       <FrameworkList Include="&lt;FileList Redist=&quot;MonoAndroid&quot; Name=&quot;Xamarin.Android Base Class Libraries&quot;&gt;" />
+        <File AssemblyName="System.Buffers" Version="4.0.99.0" />
```

Comment:
This won't do what you think it does.

Aside: the indentation here is misleading: it's *not* a child element of `<FrameworkList/>` (previous line), because the previous line ends with `/>`, not `>`.

Verification:

```
$ curl -o 'xamarin.android-oss_v9.1.99.131_Darwin-x86_64_HEAD_c3de1d65.zip' 
https://jenkins.mono-project.com/job/xamarin-android-pr-builder/4243/Azure/processDownloadRequest/xamarin-android-pr-builder-4243/xamarin-android/xamarin.android-oss_v9.1.99.131_Darwin-x86_64_HEAD_c3de1d65.zip
$ unzip xamarin.android-oss_v9.1.99.131_Darwin-x86_64_HEAD_c3de1d65.zip xamarin.android-oss_v9.1.99.131_Darwin-x86_64_HEAD_c3de1d65/bin/Debug/lib/xamarin.android/xbuild-frameworks/MonoAndroid/v1.0/RedistList/FrameworkList.xml
$ cat xamarin.android-oss_v9.1.99.131_Darwin-x86_64_HEAD_c3de1d65/bin/Debug/lib/xamarin.android/xbuild-frameworks/MonoAndroid/v1.0/RedistList/FrameworkList.xml
<FileList Redist="MonoAndroid" Name="Xamarin.Android Base Class Libraries">
</FileList>
```

(The URL in the above `curl` command is from the [**Azure Artifacts** link for your PR build](https://jenkins.mono-project.com/job/xamarin-android-pr-builder/4243/Azure/).)

The problem is what `@(FrameworkList)` is used for: https://github.com/xamarin/xamarin-android/blob/608d950a703ddb527258565c2a0705aa37f71789/src/mono-runtimes/mono-runtimes.targets#L550-L554

It's just passed as-is to the `<WriteLinesToFile/>` task.

I *believe* -- but have not tried -- that *this* is what you actually want:

```xml
    <ItemGroup>
      <FrameworkList Include="&lt;FileList Redist=&quot;MonoAndroid&quot; Name=&quot;Xamarin.Android Base Class Libraries&quot;&gt;" />
      <FrameworkList Include="  &lt;File AssemblyName=&quot;System.Buffers&quot; Version=&quot;4.0.99.0&quot; /&gt;" />
      <FrameworkList Include="  &lt;File AssemblyName=&quot;System.Memory&quot; Version=&quot;4.0.99.0&quot; /&gt;" />
      <FrameworkList Include="&lt;/FileList&gt;" />
    </ItemGroup>
```